### PR TITLE
Plugin Details: adds cta usps

### LINKED ISSUE #57759: Plugin Details: Check / add the USPs under the CTA for a free and paid plugin
Designs oU6nEIeiKSg8CayhEtgR6b-fi-6559%3A46946

- Add the USPs (unique selling point) to the plugin details page
- Refine the copy. Let's see if we can remove any billing period references so that they are more abstract and we reduce complexity

SS 2021-11-18 at 12 28 31

SS 2021-11-08 at 17 19 02

### PR DESCRIPTION
**Pull request by cpapazoglou**
#### Changes proposed in this Pull Request

This PR adds some usps under the CTA button by adding a `<USPS/>` component.

Note: I couldn't find the icons in figma in the Calypso icons library so I have swapped them with gridicons (https://automattic.github.io/gridicons/)

#### Testing instructions

|Low tier | High Tier|
|-------|------|
|SS 2021-12-28 at 16 09 36|SS 2021-12-28 at 16 09 56|

Click on the Calypso live link and run the following scenarios:

**Low Tier Plan & Free Plugin**
- Show the domain USP only for FREE sites
- Support
- Hosting

**Low Tier Plan & Paid Plugin**
- Plugin Updates
- Refund
- Show "Plan upgrade required".
- Show the domain USP only for FREE sites
- Support
- Hosting

**Premium Tier Plan & Free Plugin**
- Display nothing

**Premium Tier Plan & Paid Plugin**
- Plugin Updates
- Refund
- Support

**Jetpack**
- Display nothing

Fixes #57759

### CODE REVIEW

File: client/my-sites/plugins/plugin-details-CTA/usps.tsx [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+						eligibilities: [ 'marketplace' ],
+					},
+			  ]
+			: [] ),
+		...( isMarketplaceProduct
+			? [
+					{
+						id: 'refund',
+						image: <Gridicon icon="refund" size={ 16 } />,
+						text: translate( '30 day money-back guarantee' ),
```
Comment (a8ci18n) [inline client/my-sites/plugins/plugin-details-CTA/usps.tsx:55]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **30** times:
`translate( '30 day money back guarantee' )` *ES Score: 13*
See 2 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/plugin_cta_usps)

File: client/my-sites/plugins/plugin-details-CTA/usps.tsx [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+						eligibilities: [ 'marketplace' ],
+					},
+			  ]
+			: [] ),
+		...( shouldUpgrade
+			? [
+					{
+						id: 'plan',
+						className: 'title',
+						text: translate( 'Included in the Business plan' ),
```
Comment (a8ci18n) [inline client/my-sites/plugins/plugin-details-CTA/usps.tsx:65]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Also included in the Business Plan' )` *ES Score: 8*

File: client/my-sites/plugins/plugin-details-CTA/usps.tsx [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+						eligibilities: [ 'needs-upgrade' ],
+					},
+			  ]
+			: [] ),
+		...( shouldUpgrade || isMarketplaceProduct
+			? [
+					{
+						id: 'support',
+						image: <Gridicon icon="chat" size={ 16 } />,
+						text: translate( 'Live chat support 24x7' ),
```
Comment (a8ci18n) [inline client/my-sites/plugins/plugin-details-CTA/usps.tsx:95]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **21** times:
`translate( 'Live chat support 24X7' )` *ES Score: 14*

**Comment by cpapazoglou** [comment]
@SaxonF can you take a look here please?

**Review by gavande1 — APPROVED** [review]
I've tested following:

**Low Tier Plan & Free Plugin**

Show the domain USP only for FREE sites ✅ 
Support ✅ 
Hosting ✅ 

**Low Tier Plan & Paid Plugin**

Plugin Updates ✅ 
Refund ✅ 
Show "Plan upgrade required". ✅ 
Show the domain USP only for FREE sites ✅ 
Support ✅ 
Hosting ✅ 

**Premium Tier Plan & Free Plugin**

Display nothing ✅ 

**Premium Tier Plan & Paid Plugin**

Plugin Updates ✅ 
Refund ✅ 
Support ✅ 

**Jetpack**

Display nothing ✅ 

LGTM 👍

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/7065383

Thank you @cpapazoglou for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/7065383).

**Pull request merged**